molecule/logger.py

```diff
@@ -19,11 +19,27 @@
 #  DEALINGS IN THE SOFTWARE.
 
 import logging
+import os
 import sys
 
 import colorama
+from ansible.module_utils.parsing.convert_bool import boolean as to_bool
 
-colorama.init(autoreset=True)
+
+def should_do_markup(fh=sys.stdout):
+    py_colors = os.environ.get('PY_COLORS', None)
```

Comment:
Why not fall-back to a boolean value instead? You'd not need to do None-check at all, then.
```suggestion
    is_tty = fh.isatty() and os.environ.get('TERM') != 'dumb'
    py_colors = os.environ.get('PY_COLORS', is_tty)
    return to_bool(py_colors, strict=False)
```

Reply:
Not really: we cannot have a default boolean value here because the default behavior is to use detection. Ansible to_bool does return only booleans, it does not have any `undefined` value.

I used None as undefined.

Reply:
oh, makes sense